### PR TITLE
doc: update minimum and recommended requirements

### PR DESCRIPTION
**Pull request by simondeziel**

### CODE REVIEW

File: doc/requirements.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
 
-LXD requires Go 1.20 or higher and is only tested with the Golang compiler.
+LXD requires Go 1.21 or higher and is only tested with the Golang compiler.
 
 We recommend having at least 2GiB of RAM to allow the build to complete.
 
 ## Kernel requirements
 
-The minimum supported kernel version is 5.4.
+The minimum supported kernel version is 5.15 but older kernel may work to some degree.
```
Comment (ru-fu) [inline doc/requirements.md:13]:
```suggestion
The minimum supported kernel version is 5.15, but older kernels should also work to some degree.
```
("may" is used for permission.)

File: doc/requirements.md [hunk 2]
```diff
@@ -19,18 +19,19 @@ LXD requires a kernel with support for:
...
   ([`io_setup(2)`](https://man7.org/linux/man-pages/man2/io_setup.2.html), etc.)
 
-The following optional features also require extra kernel options:
+The following optional features also require extra kernel options or newer versions:
 
 * Namespaces (`user` and `cgroup`)
 * AppArmor (including Ubuntu patch for mount mediation)
 * Control Groups (`blkio`, `cpuset`, `devices`, `memory`, `pids` and `net_prio`)
 * CRIU (exact details to be found with CRIU upstream)
+* SKBPRIO/QFQ qdiscs (for `limits.priority`, min kernel 5.17)
```
Comment (ru-fu) [inline doc/requirements.md:28]:
```suggestion
* SKBPRIO/QFQ qdiscs (for `limits.priority`, minimum kernel 5.17)
```

Comment (ru-fu) [reply doc/requirements.md:28]:
```
QFQ
SKBPRIO
qdiscs
```
all need to go into .wordlist.txt.

File: doc/.wordlist.txt [hunk 3]
```diff
@@ -189,7 +190,10 @@ preseed
 proxied
 proxying
 PTS
+qdisc
+qdisc
```
Comment (ru-fu) [inline doc/.wordlist.txt:194]: duplicate

Comment (ru-fu) [reply doc/.wordlist.txt:194]: But you probably need "qdiscs" - aspell isn't smart when it comes to words that are defined here. :(

**Review by tomponline — APPROVED** [review]

**Pull request merged**